### PR TITLE
README.md update

### PR DESCRIPTION
**Pull request by bram-pkg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,32 @@
...
+## Usage
+
+```yaml
+version: '3'
+
+services:
+  whoami:
+    image: jwilder/whoami
+    ports:
+      - 8000:8000
```
Comment (raqbit) [inline README.md]: Maybe use `8080` like in the examples?

File: README.md [hunk 2]
```diff
@@ -5,6 +5,32 @@
...
+  
+  disrupt:
+    image: blamebutton/disrupt
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    deploy:
+      placement:
+        constraints:
+          # Disrupt needs permission to update services, only managers are allowed to do thaat.
```
Comment (raqbit) [inline README.md]: typo `that`

File: README.md [hunk 3]
```diff
@@ -23,9 +49,23 @@ Here is a small list of notificatio providers Apprise supports:
 
 ## Contributing
 
-Feel free to make a feature request or if you have Python experience; pull requests are welcome too!
+Feel free to make a feature request or if you have Python experience; pull requests are welcome
```
Comment (Raqbit) [inline README.md]: `;` -> `:`

File: README.md [hunk 4]
```diff
@@ -23,9 +49,23 @@ Here is a small list of notificatio providers Apprise supports:
...
+### Could not connect to Docker Engine
+
+Did you mount the Docker socket to the container? Check out the [example](#usage) if you want 
+to know how.
+
+In the special case that you are accessing Docker over TCP, you should place Disrupt in the 
+same network as your TCP socket. Using a Docker socket proxy (like 
+[docker-socket-proxy](https://hub.docker.com/r/tecnativa/docker-socket-proxy/), or 
+[sockguard](https://github.com/buildkite/sockguard)) is recommended for enhanced security 
+in this case.
```
Comment (raqbit) [inline README.md]: Using `docker-socket-proxy` is mainly a security measure used to guard the docker socket from malicious abuse. If the disrupt container gets compromised, having a `docker-socket-proxy` proxy between disrupt and the actual socket will prevent the attacker from executing malicious commands (e.g., spawn a crypto miner container) if the proxy is configured to only allow `GET` requests to `info` and `services`.

**Review by raqbit — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Raqbit) [reply README.md]: Also, `.` at the end :)

**Review by raqbit — APPROVED** [review]

**Pull request merged**